src/java/org/apache/cassandra/cql3/constraints/ColumnConstraint.java

```diff
@@ -89,5 +79,5 @@ public static int getSerializer(Class clazz)
      *
      * @return the Constraint type serializer
      */
-    ConstraintsSerializers getConstraintType();
+    ConstraintsSerializers getConstraintSerializerType();
```

Comment:
@bbotella 

Why not to rename `ConstrantSerializers` enum to `ConstraintType`? Then you would have a method called "`getConstraintType` which would return `ConstraintType` which makes more sense to me and reads better.

The fact that `ConstraintType` holds serializers does not mean whole enum has to be named `ConstraintSerializers`. What you have in that enum are types of constraints which have accidentally their serializers too.

Reply:
Not a super strong opinion on either of those. I choose to stay with the `ConstraintSerializers` version for that reason, the values of that enum are serializers. I'd rather keep it like this unless you really think `ConstraintType` is the way to go.

Reply:
I think it is really the way to go. You can have other usages of this enum where you have a bunch of `ColumnConstraint`'s and you want to know their types. What would you do? You would start to "instanceof"? So you call this method. But that method is called `getConstraintSerializerType`, which returns ... `ConstraintsSerializers`. Huh ..

`ContraintType` goes beyond "serializers". It just happens to carried them too.

Reply:
Fair point. Let me rename it then.